specification/src/main/asciidoc/license-efsl.adoc

```diff
@@ -55,7 +55,7 @@ The notice is:
 
 "Copyright (c) 2018,2020 Eclipse Foundation. This software or
 document includes material copied from or derived from 
-Jakarta EE Platform Specification, https://jakarta.ee/specifications/platform/9/."
+Jakarta EE Platform Specification, https://jakarta.ee/specifications/platform/9/[]."
```

Comment:
I thought that you would need link:https://jakarta.ee/specifications/platform/9/[]

Reply:
Nope.  The link: is not needed. Already verified.